### ISSUE TITLE
Display "Track Roads" as different from normal roads

### ISSUE THREAD

**Issue by hplewis** [ENT8R/streetcomplete-mapstyle#116]
**Use case**
When mapping earlier today, I saw it ask some questions about a "road" that was next to a park. In person, though, this just looked like a dirt foot path for people to walk along (i.e. trail). In the app, I wrote a note that this definitely wasn't a road. When I looked later at my note when I got to my computer, I saw it was properly labeled as an "Unmaintained Track Road".

**Proposed Solution**
I think it would be beneficial to display "Track Roads" as different from normal roads so that it's obvious there isn't a mislabeling in OSM. Maybe display them closer to a foot path, or something in between?

**Comment by timothywashere** [comment]
To be honest, I'm constantly finding roads and paths tagged as Track when they shouldn't be. If anyone was considering an option "This isn't a track" I would fully support it.

**Comment by matkoniecz** [comment]
Note that already it is tricky to distinguish roads and footways.

I have no great ideas right now how to also distinguish `highway=track`

**Comment by hplewis** [comment]
Yeah I can imagine displaying would be difficult to distinguish. I think maybe just a change where, in quests, instead of like, "What material is this road made out of?" (paraphrasing), maybe it could ask "What material is this track road made out of?"?

It's not a huge issue; I just find myself adding notes saying like "the road doesn't actually turn here, its just a path" and then realizing it's a track road.

**Comment by hplewis** [comment]
The actual path itself, I could see being a track road? Definitely not used for vehicles on any normal basis. Maybe it was at one point, but is just a walking trail now. There are entrances for vehicles if needed though. If it helps, this is the specific track road: https://www.openstreetmap.org/way/122891411

**Comment by matkoniecz** [comment]
Oh, I thought that you want to show it differently on the map.

Describing it differently may be easier to do but would put heavier burden on translators and it would be often tricky to translate.

**Comment by hplewis** [comment]
Fair enough. It's not a pressing issue, just something I thought could be improved.
